cli/pack/systemd_dir.go

```diff
@@ -125,32 +122,64 @@ func getSystemdTemplate(projectCtx *project.ProjectCtx) (templates.Template, err
 		log.Warnf(
 			"App directory doesn't contain stateboard entrypoint script `%s`. "+
 				"Stateboard systemd service unit file wouldn't be delivered",
-			project.StateboardEntrypointName,
+			projectCtx.StateboardEntrypoint,
 		)
 	}
 
 	return &systemdFilesTemplate, nil
 }
 
+func getSystemdCtx(projectCtx *project.ProjectCtx) *systemdCtx {
+	var ctx systemdCtx
+
+	ctx.Name = projectCtx.Name
+	ctx.StateboardName = projectCtx.StateboardName
+
+	ctx.DefaultWorkDir = project.GetInstanceWorkDir(projectCtx, "default")
+	ctx.InstanceWorkDir = project.GetInstanceWorkDir(projectCtx, "%i")
```

Comment:
What does "%i" mean here?

Reply:
It's `%i` from intantiated systemd unit. When you start `myapp@storage` instance, the `%i` is replaced with `storage`.

Reply:
I can use this value as a named constant and add a link to the systemd units doc.

Reply:
Well, thanks for explanation